auction/src/components/Login.js

```diff
@@ -0,0 +1,73 @@
+import React, { useState, useContext } from 'react'
+import { Link, Redirect } from 'react-router-dom';
+
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faGooglePlus, faFacebookSquare } from '@fortawesome/free-brands-svg-icons';
+
+import { UserContext } from '../services/UserContext';
+import { login } from '../services/SessionService';
+
+import TextField from './Controls/TextField';
+import Label from './Controls/Label';
+import Button from './Controls/Button';
+import Container from './Common/Container';
+import Divider from './Common/Divider';
+import FormGroup from './Controls/FormGroup';
+import Breadcrumbs from './Common/Breadcrumbs';
+
+const Login = ({ location }) => {
+    const { userData, dispatch} = useContext(UserContext);
+
+    const [email, setEmail] = useState('');
+    const [password, setPassword] = useState('');
+
+    const refferer = location.state && location.state.referrer;
+    const redirectPath = refferer ? refferer.path : '/';
+
+    return (
+        <>
+            {userData.loggedIn ?
+            <Redirect to={redirectPath} />
+            :
+            <>
+            <Breadcrumbs current='login'/>
+            <Divider/>
+            <Container title='LOGIN'>
+                <FormGroup>
+                    <Label isFor='email' label='Email' />
+                    <TextField onChange={e => setEmail(e.target.value)} name='email' placeholder='e.g. johndoe@mail.com' fullWidth />
```

Comment:
move to function above return statement.

Reply:
Done